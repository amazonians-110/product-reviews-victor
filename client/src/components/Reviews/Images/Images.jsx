/* eslint-disable no-useless-constructor */
/* eslint-disable react/prefer-stateless-function */
import React from 'react';

class Images extends React.Component {
  constructor(props) {
    super(props);
  }

  render() {
    return (
      <div>
        <b>Customer Images</b>
        <div />
        <h6>See all customer images</h6>
      </div>
    );
  }
}

export default Images;
