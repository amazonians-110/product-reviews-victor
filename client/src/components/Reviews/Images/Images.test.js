import React from 'react';
import { shallow } from 'enzyme';

import Images from './Images';

describe('Images', () => {
  it('should render correctly', () => {
    
    const component = shallow(<Images/>);

    expect(component).toMatchSnapshot();
  });
});
