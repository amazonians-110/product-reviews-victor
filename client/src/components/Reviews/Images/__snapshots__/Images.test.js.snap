// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Images should render correctly 1`] = `
<div>
  <b>
    Customer Images
  </b>
  <div />
  <h6>
    See all customer images
  </h6>
</div>
`;
